src/doc/src/reference/manifest.md

```diff
@@ -255,7 +259,7 @@ package.
 ```toml
 [package]
 # ...
-repository = "https://github.com/rust-lang/cargo/"
+repository = "https://github.com/rust-lang/cargo"
```

Comment:
Deleted trailing slash to match the real cargo.

https://github.com/rust-lang/cargo/blob/92395d90106b3b61bcb68bcf2069052c93771764/Cargo.toml#L18